ios/fastlane/Fastfile

```
@@ -3,8 +3,22 @@ default_platform(:ios)
   desc "Test build"
   lane :test do
+    require 'net/http'
+    require 'json'
     UI.message("Building for testing")
     FileUtils.remove_dir(xcarchivePath) if File.directory?(xcarchivePath)
+    
+    response=Net::HTTP.get(URI("https://itunes.apple.com/lookup?id=6744280856&country=KR"))
+    # response = `curl -s "https://itunes.apple.com/lookup?id=6744280856&country=KR"`
+    data=JSON.parse(response)
+    latest_app_store_version=data['results'][0]['version']
+    UI.message("#{latest_app_store_version}")
```

RuboCop (1.81.7)

[convention] Prefer `to_s` over string interpolation.

(Style/RedundantInterpolation)